K2Bridge.Tests.UnitTests/KustoDAL/KustoDataAccessTests.cs

```diff
@@ -308,8 +540,10 @@ public async Task ResolveIndex_WithValidInput_ReadsQueryExecutorValidDatabase(st
             mockQueryExecutor.SetupGet(x => x.DefaultDatabaseName).Returns(mockDetails.Object.DefaultDatabaseName);
             var searchString = $"search TableName: '{tableName}' | search DatabaseName: '{databaseName}' |";
             using IDataReader stubIndexReader = new DataReaderMock(
-                new List<Dictionary<string, object>>() {
-                    new Dictionary<string, object> {
+                new List<Dictionary<string, object>>()
```

Comment:
please revert indentation changes

Reply:
I reverted the line breaks which were incompatible with the code and that's most of the changes,
but the indentation changes were places that didn't fit with the code style and are now easier to read